sktime/param_est/base.py

```diff
@@ -46,19 +45,35 @@ def _coerce_to_list(obj):
 class BaseParamFitter(BaseEstimator):
     """Base parameter fitting estimator class.
 
-    The base parameter fitter specifies the methods and method signatures that all
-    parameter fitter have to implement.
-
-    Specific implementations of these methods is deferred to concrete instances.
+    This base class defines the interface for parameter estimators that accept
+    both feature data (X) and target values (y). It provides methods for fitting,
+    updating, and inspecting the fitted parameters.
+
+    Parameters
+    ----------
+    None
+
+    Attributes
+    ----------
+    _is_fitted : bool
+        Flag indicating whether the estimator has been fitted.
+    _X : sktime compatible container
+        Stored feature data from fit and update.
+    _y : array-like
+        Stored target data from fit and update.
     """
 
     # default tag values - these typically make the "safest" assumption
     _tags = {
         "object_type": "param_est",  # type of object
-        "X_inner_mtype": "pd.DataFrame",  # which types do _fit/_predict, support for X?
+        "X_inner_mtype": "pd.DataFrame",  # which types do _fit/_predict support for X?
+        "y_inner_mtype": "pd.Series",  # which types do _fit/_predict support for y?
         "scitype:X": "Series",  # which X scitypes are supported natively?
+        "scitype:y": "univariate",  # which y are fine: univariate/multivariate/both
```

Comment:
this should be the same as `scitype:X`

Reply:
A lot of the code I wrote was inspired by the entire codebase, and everywhere I looked scitype:y was either univariate, multivariate, or both

Reply:
yes

Reply:
the fields need to make sense as part of the design. In this case, consistency with `X` is important.